Sources/DoubleExtensions.swift

```diff
@@ -14,6 +14,9 @@ extension Double {
 
     /// EZSE: Converts Double to Int
     public var toInt: Int { return Int(self) }
+    
+    /// EZSE: Converts Double to CGFloat
+    public var toCGFloat: CGFloat { return CGFloat(self) }
```

Comment:
There will be loss of precision at different values on a 32bit device, which are limited to iPhone5 and before, vs on a 64-bit device

As documented in `CGBase.h`, `CGFloat` is a `typedef` for `double` or `float` depending on the device.
```
#if defined(__LP64__) && __LP64__
# define CGFLOAT_TYPE double
# define CGFLOAT_IS_DOUBLE 1
# define CGFLOAT_MIN DBL_MIN
# define CGFLOAT_MAX DBL_MAX
#else
# define CGFLOAT_TYPE float
# define CGFLOAT_IS_DOUBLE 0
# define CGFLOAT_MIN FLT_MIN
# define CGFLOAT_MAX FLT_MAX
#endif
```

How should we document it here?